src/language-js/parser-espree.js

```diff
@@ -30,8 +31,8 @@ function createParseError(error) {
 }
 
 function parse(originalText, parsers, options) {
-  const { parse, latestEcmaVersion } = require("espree");
-  parseOptions.ecmaVersion = latestEcmaVersion;
+  // @ts-ignore
```

Comment:
@sosukesuzuki Do you know how to fix this?

Reply:
I've created PR (https://github.com/prettier/prettier/pull/11171) to fix it.

Reply:
I don't understand, why it didn't fail before?

Reply:
Hmm, I don't know. However I tried to investigate and found out that if we force rewrite the value of the `main` field in `/node_modules/espree/package.json`, the error will not occur. So the reason is that the espree file structure or package.json has been changed (I don't know details...).

Reply:
Maybe because tsc don't understand `exports`